src/internal.c

```diff
@@ -12562,7 +12562,7 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                     }
 
                 #ifdef HAVE_OCSP
-                    if (doLookup && SSL_CM(ssl)->ocspEnabled) {
+                    if (ret == 0 && doLookup && SSL_CM(ssl)->ocspEnabled) {
```

Comment:
I can't see the need for this.
We check ret == 0 before entering next outer if.
Any setting of ret < 0 results in a 'goto exit_ppc'

The next one I can see.

Reply:
Agreed, I noticed that too. Pushing fix.